log/logging.go

```diff
@@ -135,19 +134,14 @@ func SetPrefix(prefix string) {
 // Fatal outputs a severe error message just before terminating the process.
 // Use judiciously.
 func Fatal(id, description string, keysAndValues ...interface{}) {
-	if Level < LevelFatal {
-		return
-	}
-	DefaultLogger.logMessage(LevelFatalName, id, description, keysAndValues...)
-	osExit(1)
+	keysAndValues = append([]interface{}{"id", id}, keysAndValues...)
```

Comment:
> This whole “id” thing is a little confusing for me

Totally agree with this. 

> let’s just use some crazy string as its key, something like @@_💩_prefix_☃_@@.

Ooh, me likey. We could just namespace it: `golog:id=<id>` maybe?